### PR TITLE
Update Ruby & Rails

### PR DESCRIPTION
**Pull request by jbirdjavi**
Add bundle-audit and brakeman

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-18.04 # use 18 because version 20 didn't have helvetica font
+    runs-on: ubuntu-latest
```
Comment (knutsenm) [inline .github/workflows/ruby.yml:11]: Hmm... tests passed. I guess the latest Ubuntu _does_ have Helvetica?

File: .github/workflows/ruby.yml [hunk 2]
```diff
@@ -45,3 +45,20 @@ jobs:
 
       - name: RSpec tests
         run: bundle exec rspec --color
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out files
+        uses: actions/checkout@v2
```
Comment (knutsenm) [inline .github/workflows/ruby.yml:53]: I thought you were moving to `actions/checkout@v3` (here and above in `test`)?

File: .github/workflows/ruby.yml [hunk 3]
```diff
@@ -45,3 +45,20 @@ jobs:
...
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Bundle audit
+        run: bundle exec bundle audit check --update --ignore CVE-2015-9284
+
+      - name: Brakeman
+        run: bundle exec brakeman -A -q --ensure-latest --no-pager
```
Comment (knutsenm) [inline .github/workflows/ruby.yml:64]: Impose `standardrb` as well, perhaps?

**Review by knutsenm — APPROVED** [review]

File: .github/workflows/ruby.yml (hunk 1 above)
Comment (jbirdjavi) [reply .github/workflows/ruby.yml:11]: I guess so. I figured I'd try it once before I looked into it. :)

File: .github/workflows/ruby.yml (hunk 2 above)
Comment (jbirdjavi) [reply .github/workflows/ruby.yml:53]: Yes, thanks.

File: .github/workflows/ruby.yml (hunk 3 above)
Comment (jbirdjavi) [reply .github/workflows/ruby.yml:64]: Sure, why not?

**Review by knutsenm — APPROVED** [review]

**Pull request merged**